scripts/utils/index.ts

```diff
@@ -59,6 +59,8 @@ export enum DEPLOYMENT_SALTS {
   MULTICHAIN_VALIDATOR_MODULE = "PROD_MULTICHAIN_VALIDATOR_MODULE_V1_22082023_vdQZbfh",
   PASSKEY_MODULE = "PROD_PASSKEY_MODULE_V1_22082023_n0nz9WE",
   SESSION_KEY_MANAGER_MODULE = "PROD_SESSION_KEY_MANAGER_MODULE_V1_22082023_o3aiYAr",
+  SESSION_KEY_MANAGER_MODULE_V2 = "PROD_SESSION_KEY_MANAGER_MODULE_V2_02092023_lgskxU1", // 0x000002fbffedd9b33f4e7156f2de8d48945e7489
```

Comment:
@ankurdubey521 here I updated salt also and used this salt to deploy updated session key managed + batched router on mumbai.

since contracts are further updated, and intermediate is really useless, we can just update the salt for the same variables and carry out deployments.

Reply:
Why did we create a new SESSION_KEY_MANAGER_MODULE_V2 instead of using the existing SESSION_KEY_MANAGER_MODULE? Also, what does intermediate refer to in this context?

Reply:
intermediate as in this current one, PROD_SESSION_KEY_MANAGER_MODULE_V2_02092023_lgskxU1
because the module is again updated.

so we will deploy SESSION_KEY_MANAGER_MODULE_V2 (some new salt and latest updated bytecode) 
only reason for calling V2 is because we have full audit for some other commit! (SESSION_KEY_MANAGER_MODULE)